skillmap/src/App.tsx

```diff
@@ -124,7 +124,9 @@ class AppImpl extends React.Component<AppProps, AppState> {
         const targetId = pxt.appTarget.id;
         const pxtBranch = pxt.appTarget.versions.pxtCrowdinBranch;
         const targetBranch = pxt.appTarget.versions.targetCrowdinBranch;
-        pxt.Util.enableLiveLocalizationUpdates();
+        if (!pxt.BrowserUtils.isLocalHostDev()) {
```

Comment:
don't need the electron check here?

Reply:
Not really as we just bounce out to the livesite for skillmap in offline app anyways, but probably should just include it anyways to save having to remember it whenever we do support skillmaps there.

```suggestion
        if (!pxt.BrowserUtils.isLocalHostDev() && !pxt.BrowserUtils.isPxtElectron()) {
```